src/styles/_Popover.scss

```diff
@@ -51,6 +51,7 @@
   text-overflow: ellipsis;
   width: 100%;
   display: block;
+  cursor: pointer;
```

Comment:
any chance you could update the styling to match the styles of the website? Font, etc.